src/evict/evict_page.c

```diff
@@ -125,8 +125,23 @@ __evict_stats_update(WT_SESSION_IMPL *session, uint8_t flags)
 }
 
 /*
+ * NO_AUTO_FORMAT
+ *
  * __wt_evict --
- *     Evict a page.
+ *     This function evicts a page from memory by taking exclusive access to the page. Based on the
+ *     page's state, the function either reconciles and writes the page to disk or simply discards
+ *     it from the cache. It can be called multiple times to evict a page by either eviction worker
```

Comment:
I'm not sure we should say "multiple times" here. It makes me think we call this function multiple times while evicting a page.

Reply:
Updated in [a79e103](https://github.com/wiredtiger/wiredtiger/pull/11096/commits/a79e103b84937154cc87b231958aed81132bd5ef)